{
    "uuid": "33afc1b3-85a9-4394-8423-fa137f30a007",
    "name": "Magento\\Framework\\Validator\\Test\\Unit\\Constraint\\Option\\CallbackTest::testGetValueException",
    "description": null,
    "descriptionHtml": null,
    "start": 1711693077030,
    "stop": 1711693077031,
    "status": "passed",
    "statusDetails": {
        "known": null,
        "muted": null,
        "flaky": null,
        "message": null,
        "trace": null
    },
    "stage": "finished",
    "steps": [],
    "attachments": [],
    "parameters": [
        {
            "name": "Data set",
            "value": "#3",
            "excluded": null,
            "mode": null
        }
    ],
    "historyId": "acec6818d361a22d00c2224f7146201b",
    "testCaseId": "44afe60acd6654163e3590b497bb5619",
    "rerunOf": null,
    "fullName": "Magento\\Framework\\Validator\\Test\\Unit\\Constraint\\Option\\CallbackTest::testGetValueException",
    "labels": [
        {
            "name": "testClass",
            "value": "Magento\\Framework\\Validator\\Test\\Unit\\Constraint\\Option\\CallbackTest"
        },
        {
            "name": "testMethod",
            "value": "testGetValueException"
        },
        {
            "name": "host",
            "value": "shashank"
        },
        {
            "name": "thread",
            "value": null
        }
    ],
    "links": []
}